src/glmfit.jl

```diff
@@ -510,6 +525,7 @@ glm(X, y, args...; kwargs...) = fit(GeneralizedLinearModel, X, y, args...; kwarg
 GLM.Link(mm::AbstractGLM) = mm.l
 GLM.Link(r::GlmResp{T,D,L}) where {T,D,L} = L()
 GLM.Link(r::GlmResp{T,D,L}) where {T,D<:NegativeBinomial,L<:NegativeBinomialLink} = L(r.d.r)
+GLM.Link(r::GlmResp{T,D,L}) where {T,D,L<:PowerLink} = r.link
```

Comment:
Isn't this enough with this PR?
```suggestion
GLM.Link(r::GlmResp) = r.link
```